### PR TITLE
Documentation for help.js

### PR DESCRIPTION
**Pull request by vaibhavgarg11**
### Description of the Change
Added comments for documentation in jsdocs and created out/ folder which has the respextive html file showing the documentation. 

### Benefits
Documentation of code

### Possible Drawbacks
Formatting of the documentation is needed

### CODE REVIEW

File: lib/help.js [hunk 1]
```diff
@@ -11,7 +11,14 @@ var helpjson = {
...
 	'help'				: 'Print help manual'
 };
+/**
+* Displays the list of autolab commands along with their functions.
+* @function help
+* @param {null}
+* @return {object} object
+*/
 module.exports = function() {
+
```
Comment (SebastinSanty) [inline lib/help.js]: Remove extra line

File: lib/help.js [hunk 2]
```diff
@@ -11,7 +11,14 @@ var helpjson = {
 	'submit'			: 'To submit the code to JavaAutolab and fetch the results',
 	'help'				: 'Print help manual'
 };
+/**
+* Displays the list of autolab commands along with their functions.
+* @function help
+* @param {null}
+* @return {object} object
+*/
```
Comment (SebastinSanty) [inline lib/help.js]: `@module lib/help`?

File: lib/help.js [hunk 3]
```diff
@@ -11,7 +11,14 @@ var helpjson = {
 	'submit'			: 'To submit the code to JavaAutolab and fetch the results',
 	'help'				: 'Print help manual'
 };
+/**
+* Displays the list of autolab commands along with their functions.
+* @function help
+* @param {null}
+* @return {object} object
```
Comment (SebastinSanty) [inline lib/help.js]: The function doesn't return anything. Is this needed?

Comment (vaibhavgarg11) [inline lib/help.js]: The module help.js returns object so I was a bit confused what to write in the return statement

File: lib/help.js (hunk 2 above)
Comment (vaibhavgarg11) [inline lib/help.js]: Should I replace @function with @module ?

File: lib/help.js (hunk 3 above)
Comment (SebastinSanty) [inline lib/help.js]: I can't see anything getting returned. It is just being called from index.js for printing the help manual.

File: lib/help.js (hunk 1 above)
Comment (prasadtalasila) [inline lib/help.js]: see the relevant JSDoc option for specifying module exports.

BTW, the function does not return anything. It only prints to the console. But the function (object) itself would be available across all other places where lib/help.js is required.

**Comment by prasadtalasila** [comment]
Since another active pull request exists with help.js included in it, I am closing this pull request.

**Pull request closed without merging**